code/app-kotlin/build.gradle.kts

```diff
@@ -79,10 +79,15 @@ dependencies {
     implementation("com.adobe.marketing.mobile:edgeidentity:$mavenEdgeIdentityVersion-SNAPSHOT") {
         exclude(group = "com.adobe.marketing.mobile", module = "core")
     }
-    implementation("com.adobe.marketing.mobile:edgeconsent:2.0.0") {
+    implementation("com.adobe.marketing.mobile:edgeconsent:3.0.0-SNAPSHOT") {
         exclude(group = "com.adobe.marketing.mobile", module = "edge")
+        exclude(group = "com.adobe.marketing.mobile", module = "core")
+    }
+    implementation("com.adobe.marketing.mobile:assurance:3.0.0-SNAPSHOT") {
+        exclude(group = "com.adobe.marketing.mobile", module = "core")
     }
-    implementation("com.adobe.marketing.mobile:assurance:2.2.1")
+
+    implementation("androidx.constraintlayout:constraintlayout:2.0.4")
```

Comment:
we need to add this for the app now, but not with Assurance 2.x because it was imported with Assurance 2.x.  Now 3.x is no longer needs that, so we need to manually import for the test app.